src/org/labkey/test/pages/assay/plate/PlateTemplateList.java

```diff
@@ -0,0 +1,45 @@
+package org.labkey.test.pages.assay.plate;
+
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.pages.LabKeyPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class PlateTemplateList extends LabKeyPage<PlateTemplateList.ElementCache>
+{
```

Comment:
Is this class and the one above, PlateDesignerPage, used anywhere?

Reply:
Added some JavaDoc explicitly identifying these to page as incomplete.